ext/crypto/Cargo.toml

```diff
@@ -10,6 +10,9 @@ readme = "README.md"
 repository.workspace = true
 description = "Web Cryptography API implementation for Deno"
 
+[features]
+include_js_files_for_snapshotting = ["deno_core/include_js_files_for_snapshotting"]
```

Comment:
Why does every crate need this feature? If the embedder enables the `include_js_files_for_snapshotting` feature on `deno_core`, that same flag will also be set for all of the `ext/` crates right? (flags are crate global, not local per crate)

Reply:
I just tried that, and while the program still compiles the resulting binary is 85Mb instead of 82Mb, suggesting that it's required.